test/test.js

```diff
@@ -3397,6 +3397,69 @@ describe('Page', function() {
       }
     });
   });
+
+  describe('JSCoverage', function() {
+    it('should work', async function({page, server}) {
+      await page.coverage.startJSCoverage();
+      await page.goto(server.PREFIX + '/jscoverage/simple.html');
+      const coverage = await page.coverage.stopJSCoverage();
+      expect(coverage.length).toBe(1);
+      expect(coverage[0].url).toContain('/jscoverage/simple.html');
+      expect(coverage[0].ranges).toEqual([
+        { startOffset: 0, endOffset: 17 },
```

Comment:
Considering that you have just created a file format, I'd feel safer with some large golden-file tests.

Reply:
Added a golden file, though not a very large one.